core-web/libs/sdk/react/src/lib/deprecated/hooks/useDotcmsEditor.ts

```diff
@@ -55,7 +56,9 @@ export const useDotcmsEditor = ({ pageContext, config }: DotcmsPageProps) => {
             return;
         }
 
-        const { unsubscribe } = createUVESubscription('changes', () => onReload());
+        const { unsubscribe } = createUVESubscription(UVEEventType.CONTENT_CHANGES, () =>
```

Comment:
Should we make this accept `changes` too? The standard for subscriptions usually is a plain string

```suggestion
        const { unsubscribe } = createUVESubscription(UVEEventType.CONTENT_CHANGES, () =>
```

Reply:
We can do it as the enum or string, that can work

Also remember to expose the Enum to the developers

Reply:
We can re-check that in the next ticket